### ISSUE TITLE
How to convert a list into a string?

### ISSUE THREAD

**Issue by AshishNamdev** [AshishNamdev/iqa#16]
How to convert a list into a string?

- When we want to convert a list into a string, we can use the `.join()` method which joins all the elements into one and returns as a string.

### Example:
```python
weekdays = [‘sun’,’mon’,’tue’,’wed’,’thu’,’fri’,’sat’]
listAsString = ‘ ‘.join(weekdays)
print(listAsString)
```

**Comment by AshishNamdev** [comment]
Included in [Q12](../blob/master/questions/Q1-50.md#Q12)

**Comment by AshishNamdev** [comment]
included in commit 0ee53c12045d5ea4788e9b4527a052dd89c4cfdd
